.kitchen.yml

```diff
@@ -10,30 +10,15 @@ verifier:
   name: inspec
 
 platforms:
-  - name: osx-1010
+  - name: hbsmith/awslinux
```

Comment:
Who's this `hbsmith` character?

Reply:
This is to test Amazon 2018.03 images with Vagrant.

https://app.vagrantup.com/hbsmith/boxes/awslinux

Reply:
Sure… just wondering who that provider is. Just some random person who created a Vagrant box out of an Amazon Linux AMI?